docs/2021-lust/github_integration.md

```diff
@@ -77,13 +77,13 @@ a project that aims to *automate* tedious software installation procedures.
 
 At the end of 2015 efforts were made to tackle this issue by implementing
 GitHub integration features in EasyBuild, which automate the contribution
-workflow by running `git` commands and interacting with the [GitHub API](https://developer.github.com/v3/).
+workflow by running `git` commands and interacting with the [GitHub API](https://blogs.oracle.com/linux/post/attack-of-the-cosmic-rays).
```

Comment:
@klust Also wrong replacement here

Reply:
Should also be corrected in the latest commit.